util/util.go

```diff
@@ -39,6 +41,13 @@ const (
 	KeyServiceSecretEnv    = "KEY_SERVICE_SECRET"
 	KeyServiceSecretHeader = "key-service-secret"
 	KeyServiceURLEnv       = "KEY_SERVICE_URL"
+
+	recoveryKeyArgonTime       = 1
```

Comment:
@claucece to double-check these parameters

Reply:
yes! it is see here: https://github.com/golang/crypto/blob/master/argon2/argon2.go#L76 and also: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-argon2-03#section-9.4

In 'recoveryKeyArgonThreads' as 4, I assume we have 4CPUs available, right @DJAndries ? If we are using more threads, we can do:

```
// 4 threads x 64 MiB = 256 MiB
key := argon2.IDKey(password, salt, 1, 256*1024, 4, 32)
```

Reply:
each server pod runs on 0.5 vCPU. do you think it would be safe to reduce the thread count to 1? if not, no worries. i'm not too concerned about performance issues given the expected infrequency of recovery code usage.

Reply:
yes! we can ;)